ThirdParty/dojo-release-1.10.4/dijit/icons/commonIcons_rtl.css

```diff
@@ -119,3 +119,17 @@ The 16 x 16px icons in these sprites are action and object type images which can
 .dijitRtl .dijitDisabled .dijitIconError {
 	background-image: url('images/commonIconsObjActDisabled_rtl.png'); /* Contains both object and action icons as a sprite image for the disabled state. These would be used by buttons and menus.   */
 }
+
+.gitHubIcon {
```

Comment:
This file is part of Dojo, which is a third-party library.  Do not modify it.  There's probably a .css file in the `Apps/Sandcastle` directory you can use.  If not, @emackey will know.